---
layout: default
title: "BIO405: Computational Biology"
course_description: "An introduction to the use of computers to automate data analysis or model hypotheses in the field of biology, and its application for molecular and cellular biology, biochemistry, neuroscience, evolution, population biology, and behavior."
next: ../Unit03
previous: ../Unit01
---
**Unit 2: Computational Molecular Biology** <span id="2"></span> 
*This unit will deal with molecular biology.  From determining the
genetic sequence in a scrap of DNA to creating a 3-D virtual model of a
protein, scientists use computer modeling in this field to help us
better understand a world far beyond the reaches of the microscope.
 Study in molecular biology yields vast amounts of data that must be
carefully analyzed and interconnected to construct a realistic picture
of this world.  Computers greatly reduce the manual workload and error
associated with complex analysis.  In this unit, you will learn how to
use computers in order to better understand DNA sequencing,
transcriptional regulation, protein structure, and protein interactions.
 Upon completion of this unit, you should understand each process and be
able to create your own models based upon the principles and procedures
studied.*

**Unit 2 Time Advisory**  
This unit should take approximately 46 hours to complete.  
  
 ☐    Subunit 2.1: 1 hour  
  
 ☐    Subunit 2.2: 18 hours

  
 ☐    Introduction: 0.5 hours  
  
 ☐    Subunit 2.2.1: 0.5 hours  
  
 ☐    Subunit 2.2.2: 4.5 hours  
  
 ☐    Subunit 2.2.3: 10.5 hours  
  
 ☐    Subunit 2.2.4: 2 hours

  
 ☐    Subunit 2.3: 10 hours

  
 ☐    Subunit 2.3.1: 3 hours  
  
 ☐    Subunit 2.3.2: 1 hour  
  
 ☐    Subunit 2.3.3: 6 hours

  
 ☐    Subunit 2.4: 17 hours

  
 ☐    Subunit 2.4.1: 6 hours  
  
 ☐    Subunit 2.4.2: 11 hours

**Unit2 Learning Outcomes**  
Upon completion of this unit, the student will be able to:
-   Describe basic molecular principles.
-   Describe methods for collecting and analyzing DNA data.
-   Describe methods for creating phylogenetic trees.
-   Describe how statistical modeling is used with data on proteins.
-   Provide a qualitative explanation of Markovian models.
-   Describe methods for collecting and analyzing data on protein
    structure.

**2.1 Mathematical Treatment of Molecular Biology** <span
id="2.1"></span> 
**2.1.1 Principles of Molecular Biology** <span id="2.1.1"></span> 
-   **Reading: PDFgeni.org’s “Basic Principles of Molecular Biology 1”**
    The Saylor Foundation does not yet have materials for this portion
    of the course. If you are interested in contributing your content to
    fill this gap or aware of a resource that could be used here, please
    submit it here.

    [Submit Materials](/contribute/)

**2.2 DNA Sequencing** <span id="2.2"></span> 
-   **Reading: Science Daily News’ “A New Read on DNA Sequencing”**
    Link: Science Daily News’ [“A New Read on DNA
    Sequencing”](http://www.sciencedaily.com/releases/2010/11/101114161943.htm)
    (HTML)  
      
     Instructions: Read this article on advances in DNA sequencing.
     There are other related articles on this topic on the main page
    that you may wish to explore for further information on the
    subject.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.1 DNA Structure and Classification** <span id="2.2.1"></span> 
-   **Reading: University of Arizona: Richard Hallick’s “Introduction to
    DNA Structure”**
    Link: University of Arizona: Richard Hallick’s [“Introduction to DNA
    Structure”](http://www.blc.arizona.edu/molecular_graphics/dna_structure/dna_tutorial.html)
    (HTML)  
      
     Instructions: Please read the linked material; it provides a good
    introduction to the structure of DNA and the formation of the double
    helix.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.2 Data Collection** <span id="2.2.2"></span> 
**2.2.2.1 DNA Microarrays** <span id="2.2.2.1"></span> 
-   **Web Media: YouTube: Proneural’s “DNA Microarrays”**
    Link: YouTube: Proneural’s [“DNA
    Microarrays”](http://www.youtube.com/watch?v=VNsThMNjKhM)
    (YouTube)  
      
     Instructions: Please watch this video, which provides an example of
    DNA microarrays in plants.  Try and break down the material in to
    steps involved in the process.  
      
     Watching this video and taking notes should take approximately 15
    minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.2.2 DNA Sequencing Process** <span id="2.2.2.2"></span> 
**2.2.2.2.1 Manual** <span id="2.2.2.2.1"></span> 
-   **Reading: National Diagnostics’ “Manual Sequencing”**
    Link: National Diagnostics’ [“Manual
    Sequencing”](https://www.nationaldiagnostics.com/electrophoresis/article/manual-sequencing)
    (HTML)  
      
     Instructions: This is a brief introduction to manual sequencing.
     Two versions of sequencing (Sanger Sequencing and Maxam & Gilbert
    Sequencing) are presented on the pages following the links provided.
     Take note of the process and the differences between the two
    methods.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.2.2.2 Automated** <span id="2.2.2.2.2"></span> 
-   **Web Media: Cold Spring Harbor Laboratory: DNA Learning Center’s
    “Cycle Sequencing”**
    Link: Cold Spring Harbor Laboratory: DNA Learning Center’s [“Cycle
    Sequencing”](http://www.dnalc.org/resources/animations/cycseq.html)
    (Flash)  
      
     Instructions: Use the “continue” button to work through the
    animation on the process of cycle sequencing.  How is this process
    different from manual sequencing above?  
      
     Viewing this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.2.3 Alignment** <span id="2.2.2.3"></span> 
**2.2.2.3.1 Pairwise Alignment** <span id="2.2.2.3.1"></span> 
-   **Reading: University of Wisconsin: Mark Craven’s “Pairwise
    Alignment”**
    Link: University of Wisconsin: Mark Craven’s [“Pairwise
    Alignment”](http://www.biostat.wisc.edu/bmi576/lectures/) (PDF)  
      
     Instructions: Click on the “pairwise-alignment-1.pdf” link to
    download the PDF.  This PowerPoint will introduce you to alignments,
    edits, and gaps and define the concept of pairwise alignment.  The
    PowerPoint also connects sequence alignment to dynamic computer
    programming.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.2.3.2 Multiple Sequence Alignment** <span id="2.2.2.3.2"></span> 
-   **Reading: Wikipedia’s “Multiple Sequence Alignment”**
    Link: Wikipedia’s [“Multiple Sequence
    Alignment”](http://en.wikipedia.org/wiki/Multiple_sequence_alignment)
    (HTML)  
      
     Instructions: The reading describes multiple sequence alignments
    and how they relate to dynamic programming.  You should also view
    the picture of a multiple sequence alignment found on the same
    page.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.2.3.3 Whole Genome Alignment** <span id="2.2.2.3.3"></span> 
-   **Reading: Universität des Saarlandes: Center for Bioinformatics’
    “Whole Genome Alignment”**
    Link: Universität des Saarlandes: Center for Bioinformatics’ [“Whole
    Genome
    Alignment”](http://gepard.bioinformatik.uni-saarland.de/old_html/html/BioinformatikIIIWS0304-Dateien/)
    (PDF or PPT)  
      
     Instructions: Click on the “V4-Alignment.pdf” or “V4-Alignment.ppt”
    link to download the presentation.  This presentation explains why
    whole genome alignments are used, presents a few different ways of
    aligning sequences, and addresses local versus global alignments.
     For information in the presentation that needs more clarification,
    refer directly to the citations at the bottom of each slide.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3 Comparisons** <span id="2.2.3"></span> 
**2.2.3.1 Comparison with Sequence Databases** <span
id="2.2.3.1"></span> 
**2.2.3.1.1 FASTA** <span id="2.2.3.1.1"></span> 
-   **Reading: State University of New York – Stony Brook: Steven
    Skiena’s “FASTA”**
    Link: State University of New York – Stony Brook: Steven Skiena’s
    [“FASTA”](http://www.cs.sunysb.edu/~skiena/648/presentations/blast/FASTA.html)
    (HTML)  
      
     Instructions: The resource describes FASTA and presents a
    step-by-step process for using it.  FASTA is the first widely used
    method of using a known sequence to search a database for similar
    sequences.  The best approach for understanding this is to actually
    create a fictitious sequence and use the search engine to analyze
    it.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3.1.2 BLAST** <span id="2.2.3.1.2"></span> 
-   **Reading: National Center for Biotechnology Information’s “BLAST”**
    Link: National Center for Biotechnology Information’s
    [“BLAST”](http://blast.ncbi.nlm.nih.gov/Blast.cgi) (HTML)  
      
     Instructions: The link will take you to the BLAST website, where
    you will find a description of its capabilities and be able to use
    this search tool.  Please try the search tool to see how it
    performs.  
      
     Studying this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3.1.3 Finding Sequence Motifs** <span id="2.2.3.1.3"></span> 
-   **Reading: Nature Biotechnology: Patrik D'haeseleer’s “What Are DNA
    Sequence Motifs?”**
    Link: Nature Biotechnology: Patrik D'haeseleer’s [“What Are DNA
    Sequence
    Motifs?”](http://www.nature.com/nbt/journal/v24/n4/full/nbt0406-423.html)
    (HTML)  
      
     Instructions: This article discusses varies topics related to
    sequence motifs, including logos and searching for novel sites.  You
    should make note of key terms and use Google or a similar search
    engine to identify unknown terms in the reading.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3.2 Genome Comparison** <span id="2.2.3.2"></span> 
-   **Reading: ScienceDaily News’ “Multi-Species Genome Comparison Sheds
    New Light On Evolutionary Processes, Cancer Mutations”**
    Link: ScienceDaily News’ [“Multi-Species Genome Comparison Sheds New
    Light On Evolutionary Processes, Cancer
    Mutations”](http://www.sciencedaily.com/releases/2005/07/050727062750.htm)
    (HTML)  
      
     Instructions: The article summarizes the findings of recent work
    comparing genome sequences from multiple species.  Use this example
    as a basis for identifying other uses of multi species genome
    comparisons.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3.3 Molecular Phylogenetic Analysis** <span id="2.2.3.3"></span> 
-   **Reading: Wikipedia’s “Molecular Phylogenetics”**
    Link: Wikipedia’s [“Molecular
    Phylogenetics”](http://en.wikipedia.org/wiki/Molecular_phylogenetics)
    (HTML)  
      
     Instructions: This article describes the history, techniques,
    theoretical background, and limitations associated with Molecular
    Phylogenetics.  Familiarize yourself with any unfamiliar terms in
    the reading by using the links provided.  
      
     Reading this material should take approximately 45 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: Proceedings of the National Academy of Sciences: Michael
    A. Thomas et al’s “Molecular Phylogenetic Analysis of Evolutionary
    Trends in Stonefly Wing Structure and Locomotor Behavior”**
    Link: Proceedings of the National Academy of Sciences: Michael A.
    Thomas et al’s [“Molecular Phylogenetic Analysis of Evolutionary
    Trends in Stonefly Wing Structure and Locomotor
    Behavior”](http://www.pnas.org/content/97/24/13178.abstract) (PDF)  
      
     Instructions: At the PNAS webpage, click on the PDF link to see the
    PDF version of the file.  Read this paper and pay attention to the
    phylogenetic methods that are used to address the hypotheses
    presented in the paper.  This paper is a good example of how
    molecular phylogenetics can be used and should be viewed as one of
    many ways to apply phylogenetic methods.  
      
     Reading this material should take approximately 2 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3.3.1 Principles of Molecular Evolution** <span
id="2.2.3.3.1"></span> 
-   **Reading: Proceedings of the National Academy of Sciences: Motoo
    Kimura and Tomoko Ohta’s “On Some Principles Governing Molecular
    Evolution”**
    Link: Proceedings of the National Academy of Sciences: Motoo Kimura
    and Tomoko Ohta’s [“On Some Principles Governing Molecular
    Evolution”](http://www.pnas.org/content/71/7/2848.short) (PDF)  
      
     Instructions: Click on the PDF link to see the entire article.  The
    abstract lists five principles associated with molecular evolution.
     This is a classic article and you should understand and be able to
    list the five principles that are presented.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3.3.2 Methods for Determining Distance** <span
id="2.2.3.3.2"></span> 
**2.2.3.3.2.1 Average Linkage Clustering** <span
id="2.2.3.3.2.1"></span> 
-   **Reading: Statistics.com’s “Average Linkage Clustering”**
    Link: Statistics.com’s [“Average Linkage
    Clustering”](http://www.statistics.com/index.php?page=glossary&term_id=714)
    (HTML)  
      
     Instructions: This page provides a short paragraph about linkage
    clustering and a link for the linkage function that you should click
    on and view that information.  Use the additional links to provide a
    more complete understanding of how this function is used and where
    it can be applied.  
      
     This reading should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3.3.2.2 Neighbor-Joining** <span id="2.2.3.3.2.2"></span> 
-   **Reading: Christian de Duve Institute of Cellular Pathology: Fred
    R. Opperdoes’ “The Neighbor-Joining Method”**
    Link: Christian de Duve Institute of Cellular Pathology: Fred R.
    Opperdoes’ [“The Neighbor-Joining
    Method”](http://www.icp.ucl.ac.be/~opperd/private/neighbor.html)
    (HTML)  
      
     Instructions: The webpage includes a description of the method,
    advantages and disadvantages, and an example of how it works.  After
    reading this material, compare it to other distance-based methods
    used in phylogenetic trees such as maximum parsimony.  
      
     Reading this material should take approximately 45 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3.3.3 Maximum Parsimony** <span id="2.2.3.3.3"></span> 
-   **Reading: Christian de Duve Institute of Cellular Pathology: Fred
    R. Opperdoes’ “Maximum Parsimony”**
    Link: Christian de Duve Institute of Cellular Pathology: Fred R.
    Opperdoes’ [“Maximum
    Parsimony”](http://www.icp.ucl.ac.be/~opperd/private/parsimony.html)
    (HTML)  
      
     Instructions: The webpage demonstrates how parsimony works and ends
    with some summary notes on the subject.  Learn this material in
    order to compare it to other distance-based methods used in
    phylogenetic trees such as neighbor joining.  
      
     Reading this material should take approximately 45 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.3.3.4 Maximum Likelihood** <span id="2.2.3.3.4"></span> 
-   **Reading: University of Alaska Fairbanks: Mark Lindberg’s “Maximum
    Likelihood Estimation”**
    The Saylor Foundation does not yet have materials for this portion
    of the course. If you are interested in contributing your content to
    fill this gap or aware of a resource that could be used here, please
    submit it here.

    [Submit Materials](/contribute/)

**2.2.4 Statistical Modeling** <span id="2.2.4"></span> 
**2.2.4.1 Markov Models** <span id="2.2.4.1"></span> 
-   **Reading: Simon Fraser University: Bertille Antoine’s “Markov
    Chains”**
    The Saylor Foundation does not yet have materials for this portion
    of the course. If you are interested in contributing your content to
    fill this gap or aware of a resource that could be used here, please
    submit it here.

    [Submit Materials](/contribute/)

**2.2.4.2 Hidden Markov Models** <span id="2.2.4.2"></span> 
-   **Reading: Monash University: Lloyd Allison’s “Hidden Markov
    Models”**
    Link: Monash University: Lloyd Allison’s [“Hidden Markov
    Models”](http://www.csse.monash.edu.au/~lloyd/tildeMML/Structured/HMM/)
    (HTML)  
      
     Instructions: This reading covers different types of Markov models
    and provides an example of a Hidden Markov model.  This reading will
    help reinforce what you learned in subunit 2.2.4.1.  How are hidden
    Markov models different from standard Markov models?  Where would
    you apply them?  
      
     Reading this material should take approximately 45 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.2.4.3 Bootstrap Analysis** <span id="2.2.4.3"></span> 
-   **Reading: Christian de Duve Institute of Cellular Pathology: Fred
    R. Opperdoes’ “Bootstrapping”**
    Link: Christian de Duve Institute of Cellular Pathology: Fred R.
    Opperdoes’
    [“Bootstrapping”](http://www.icp.ucl.ac.be/~opperd/private/bootstrap.html)
    (HTML)  
      
     Instructions: This website describes the process of bootstrapping
    and explains how it is used in phylogenetic analysis.  Some useful
    samples are provided.  Make sure and understand the method and why
    it is used.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.3 Transcriptional Regulation** <span id="2.3"></span> 
**2.3.1 Transcriptional Regulation Process** <span id="2.3.1"></span> 
-   **Reading: Wikipedia’s “Transcriptional Regulation”**
    Link: Wikipedia’s [“Transcriptional
    Regulation”](http://en.wikipedia.org/wiki/Transcriptional_regulation)
    (HTML)  
      
     Instructions: The page provides a good overview of transcription
    regulation.  It also identifies some of the major differences
    between prokaryotes and eukaryotes.  How is transcriptional
    regulation more complex in eukaryotes?  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.3.1.1 Simple Regulation** <span id="2.3.1.1"></span> 
-   **Reading: National Center for Biotechnology Information: Griffiths
    et al’s “Regulation of Gene Expression”**
    Link: National Center for Biotechnology Information: Griffiths et
    al’s [“Regulation of Gene
    Expression”](http://www.ncbi.nlm.nih.gov/pubmed/8736767) (HTML)  
      
     Instructions: The reading has several sections that describe
    different elements involved in the regulation of gene expression.
     Take note of the different regulatory elements and how they are
    involved in gene expression.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.3.1.2 Regulatory Networks** <span id="2.3.1.2"></span> 
-   **Reading: French National Institute for Research in Computer
    Science and Control: Hidde de Jong’s “Mathematical Modeling of
    Regulatory Networks”**
    Link: French National Institute for Research in Computer Science and
    Control: Hidde de Jong’s [“Mathematical Modeling of Regulatory
    Networks”](http://www-sop.inria.fr/comore/arcgdyn/28fev/) (PDF)  
      
     Instructions: Click on the “arc-03-intro.pdf” link to download the
    PDF.  This reading provides a mathematical approach to modeling
    networks.  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.3.2 Data Collection** <span id="2.3.2"></span> 
**2.3.2.1 Finding Regulatory Sequence in DNA** <span
id="2.3.2.1"></span> 
-   **Reading: Nature.com: Nature Immunology Journal’s “Finding
    Regulatory Sequences”**
    Link: Nature.com: Nature Immunology Journal’s [“Finding Regulatory
    Sequences”](http://www.nature.com/ni/journal/v5/n8/extref/ni0804-768-S1.htm)
    (HTML)  
      
     Instructions: Click on the tabs while reading and performing the
    operations that are suggested.  This tutorial will give you a
    firsthand demonstration of how to find sequence motifs.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.3.3 Statistical Modeling** <span id="2.3.3"></span> 
**2.3.3.1 Bayesian Networks** <span id="2.3.3.1"></span> 
-   **Reading: Microsoft Research: David Heckerman’s “A Tutorial on
    Learning with Bayesian Networks”**
    Link: Microsoft Research: David Heckerman’s [“A Tutorial on Learning
    with Bayesian
    Networks”](http://research.microsoft.com/apps/pubs/default.aspx?id=69588)
    (PDF)  
      
     Instructions: Click on the PDF link to download the file.  This is
    a lengthy reading that introduces Bayesian networks and Bayesian
    statistics and terminology.  You should be able to describe what
    Bayesian networks are and how they can be used.  You should also
    have at least a basic understanding of the math behind them.  
      
     Reading this material should take approximately 2 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.3.3.2 Boolean Networks** <span id="2.3.3.2"></span> 
-   **Reading: Connexions: Ewa Paszek’s “Boolean Networks”**
    Link: Connexions: Ewa Paszek’s [“Boolean
    Networks”](http://cnx.org/content/m12394/latest/) (HTML)  
      
     Introductions: The reading relates Boolean networks to molecular
    biology and provides an example of how they are used in biology.
     You should able to describe what a Boolean network is and apply to
    the example given of cell cycle regulation.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.3.3.3 Dynamic Modeling** <span id="2.3.3.3"></span> 
-   **Reading: Emory University: Peter Thompson’s “Introduction to
    Dynamic Programming”**
    Link: Emory University: Peter Thompson’s [“Introduction to Dynamic
    Programming”](https://sites.google.com/site/pt9res/home/Emory-teaching/dynamic-modeling)
    (PDF)  
      
     Instructions: Click on the “Chapter 4: Introduction to Dynamic
    Programming” link to download the PDF.  The reading covers
    deterministic and stochastic dynamic programming.  You should be
    able to compare and contrast the two types of programming.  
      
     Reading this material should take approximately 2 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.3.3.4 Using Differential Equations** <span id="2.3.3.4"></span> 
-   **Reading: Lamar University: Paul Dawkins’ “Differential
    Equations”**
    Link: Lamar University: Paul Dawkins’ [“Differential
    Equations”](http://tutorial.math.lamar.edu/Classes/DE/DE.aspx)
    (HTML)  
      
     Instructions: This useful webpage defines all terms associated with
    differential equations and includes the math associated with
    transforms and first and second order equations.  Where can
    differential equations be applied in biology?  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4 Proteins** <span id="2.4"></span> 
**2.4.1 Protein Structure** <span id="2.4.1"></span> 
**2.4.1.1 Protein Structure and Classification** <span
id="2.4.1.1"></span> 
-   **Reading: Wikipedia’s “Structural Classification of Proteins”**
    Link: Wikipedia’s [“Structural Classification of
    Proteins”](http://en.wikipedia.org/wiki/Structural_Classification_of_Proteins)
    (HTML)  
      
     Instructions: The reading covers basic protein classes and links to
    further reading about each.  Make sure you understand the different
    classes of proteins and be able to provide examples for each class.
     You are strongly encouraged to explore the links and read further
    about different types of proteins.  
      
     Reading this material should take approximately 45 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.1.2 Data Collection** <span id="2.4.1.2"></span> 
**2.4.1.2.1 Protein Preparation** <span id="2.4.1.2.1"></span> 
-   **Reading: Wikipedia’s “Protein Purification”**
    Link: Wikipedia’s [“Protein
    Purification”](http://en.wikipedia.org/wiki/Protein_purification)
    (HTML)  
      
     Instructions: The page covers methods of extraction and outlines
    the steps of protein purification.  After reading this, you should
    be able to describe the steps necessary for extracting and purifying
    proteins.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.1.2.2 Protein Imaging** <span id="2.4.1.2.2"></span> 
**2.4.1.2.2.1 X-Ray Crystallography** <span id="2.4.1.2.2.1"></span> 
-   **Reading: Molecular Station’s “X-ray Crystallography”**
    Link: Molecular Station’s [“X-ray
    Crystallography”](https://web.archive.org/web/20130815015142/http://www.molecularstation.com/molecular-biology-techniques/x-ray-crystallography/)
    (HTML)  
      
     Instructions: Read this explanation of how x-ray crystallography is
    used to identify protein structure.  Be able to describe the process
    and why it can provide an image of protein structure.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.1.2.2.2 NMR** <span id="2.4.1.2.2.2"></span> 
-   **Lecture: Nobelprize.org: Kurt Wüthrich’s “NMR Studies of Structure
    and Function of Biological Macromolecules”**
    Link: Nobelprize.org: Kurt Wüthrich’s [“NMR Studies of Structure and
    Function of Biological
    Macromolecules”](http://nobelprize.org/nobel_prizes/chemistry/laureates/2002/wuthrich-lecture.html)
    (Flash)  
      
     Instructions: This is a 55 minute lecture involving Wuthrich’s
    Nobel-Prize-winning material on NMR and molecular biology.  Pay
    particular attention to the process of NMR and how it differs from
    other methods that contribute to our understanding of protein
    structure.  
      
     Viewing this lecture should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.1.3 Structure Modeling** <span id="2.4.1.3"></span> 
-   **Reading: New York University’s Bonneau Laboratory: Richard Bonneau
    and David Baker’s “Ab Initio Protein Structure Prediction”**
    Link: New York University’s Bonneau Laboratory: Richard Bonneau and
    David Baker’s [“Ab Initio Protein Structure
    Prediction”](http://chagall.med.cornell.edu/BioinfoCourse/PDFs/Lecture5/)
    (PDF)  
      
     Instructions: Click on the link entitled “Bonneau.pdf” to download
    the PDF.  This reading covers 2.4.1.3.1.1–2.4.1.3.1.4.  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.1.3.1 Structure Prediction** <span id="2.4.1.3.1"></span> 
**2.4.1.3.1.1 Homology Modeling** <span id="2.4.1.3.1.1"></span> 
*Note: This subunit is covered by the reading under subunit 2.4.1.3:
Structure Modeling.*

**2.4.1.3.1.2 Fold Recognition** <span id="2.4.1.3.1.2"></span> 
*Note: This subunit is covered by the reading under subunit 2.4.1.3:
Structure Modeling.*

**2.4.1.3.1.3 Threading** <span id="2.4.1.3.1.3"></span> 
*Note: This subunit is covered by the reading under subunit 2.4.1.3:
Structure Modeling.*

**2.4.1.3.1.4 Ab Initio** <span id="2.4.1.3.1.4"></span> 
*Note: This subunit is covered by the reading under subunit 2.4.1.3:
Structure Modeling.*

**2.4.1.3.2 Computational Protein Design** <span id="2.4.1.3.2"></span> 
-   **Reading: ScienceDirect: Arthur G. Street and Stephen L. Mayo’s
    “Computational Protein Design”**
    Link: ScienceDirect: Arthur G. Street and Stephen L. Mayo’s
    [“Computational Protein
    Design”](http://www.cell.com/structure/retrieve/pii/S0969212699800628)
    (PDF)  
      
     Instructions: Click on the PDF link on the right side of the page
    to access and read this article, which presents different
    considerations involved in the design of proteins.  It may be useful
    to outline the key points throughout the paper or make a list of
    things to consider when designing proteins.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.1.3.3 Statistical Tests of Structure Accuracy** <span
id="2.4.1.3.3"></span> 
-   **Reading: Birkbeck College, University of London: David S. Moss,
    Ian J. Tickle, and Roman Laskowski’s “Estimation of Precision and
    Accuracy in Protein Structure Refinement from X-ray Data”**
    Link: Birkbeck College, University of London: David S. Moss, Ian J.
    Tickle, and Roman Laskowski’s [“Estimation of Precision and Accuracy
    in Protein Structure Refinement from X-ray
    Data”](http://people.cryst.bbk.ac.uk/~ubcg05m/precgrant.html)
    (HTML)  
      
     Instructions: Read this proposal on protein structure accuracy.
     The reading provides some background on methods of determining
    accuracy in protein structure and also provides a list of useful
    references.  It may be useful to go directly to the references cited
    in the paper to gain more insight into statistical tests for
    accuracy.  Note that the references may not be freely accessible.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2 Protein-Protein Interactions** <span id="2.4.2"></span> 
**2.4.2.1 Principles of Protein Interactions** <span
id="2.4.2.1"></span> 
-   **Reading: National Center for Biotechnology Information: Susan
    Jones and Janet M. Thornton’s “Principles of Protein-Protein
    Interactions”**
    Link: National Center for Biotechnology Information: Susan Jones and
    Janet M. Thornton’s [“Principles of Protein-Protein
    Interactions”](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC40170/)
    (PDF)  
      
     Instructions: Download the full-text PDF of this article at the
    bottom of the page.  This reading reviews protein-protein
    interactions with respect to cell signaling.  It would be useful to
    make a list of examples of how proteins interact with other
    proteins.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2.2 Methods for Detecting Interactions** <span
id="2.4.2.2"></span> 
-   **Reading: Darmstadt University of Technology: Frank Krause’s
    “Detection and Analysis of Protein-Protein Interactions”**
    Link: Darmstadt University of Technology: Frank Krause’s [“Detection
    and Analysis of Protein-Protein
    Interactions”](http://nucleus.img.cas.cz/kurzprf/clanky_pdf_2007/)
    (PDF)  
      
     Instructions: Click on the link entitled “07\_1\_4artB.pdf” to view
    the PDF.  This is a detailed reading that presents a number of
    methods for detecting protein interactions.  Be able to describe
    several of the methods that are presented in the paper.  
      
     Reading this material should take approximately 2 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2.2.1 Gel Electrophoresis** <span id="2.4.2.2.1"></span> 
-   **Reading: The American Electrophoresis Society: Reiner Westermeier
    and Robert Marchmont’s “Blue Native Gel Electrophoresis”**
    Link: The American Electrophoresis Society: Reiner Westermeier and
    Robert Marchmont’s [“Blue Native Gel
    Electrophoresis”](http://www.aesociety.org/areas/bng_electrophoresis.php)
    (HTML)  
      
     Instructions: The article describes a method of detecting
    protein-protein interactions using Blue Native PAGE.  You should be
    able to accurately describe this method of detecting protein
    interactions.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2.2.2 Affinity Chromatography** <span id="2.4.2.2.2"></span> 
-   **Reading: Amersham Pharmacia Biotech’s “Affinity Chromatography”**
    Link: Amersham Pharmacia Biotech’s “[Affinity
    Chromatography”](http://teachline.ls.huji.ac.il/72682/reading-1.html#purif)
    (PDF)  
      
     Instructions: Click the first link under “Affinity Chromatography”
    to download the PDF.  This is a large and comprehensive manual on
    the process of Affinity Chromatography.  You should be able to
    describe the general method including how it works and any drawbacks
    to the method.  
      
     Reading this material should take approximately 2 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2.2.3 Mass Spectroscopy** <span id="2.4.2.2.3"></span> 
-   **Reading: University of Arizona: Daniel Figeys, Linda D. McBroom,
    and Michael F. Moran’s “Mass Spectrometry for the Study of
    Protein-Protein Interactions”**
    Link: University of Arizona: Daniel Figeys, Linda D. McBroom, and
    Michael F. Moran’s [“Mass Spectrometry for the Study of
    Protein-Protein
    Interactions”](http://www.chem.arizona.edu/classes/bioc471/pages/Lecture23/Lecture23.html)
    (PDF)  
      
     Instructions: Click on the link entitled “Protein Interactions” to
    download the PDF.  The main page also contains information about
    mass spectrometry and genomics that may be of interest to you as you
    study.  You should be able to describe the general method including
    how it works and any drawbacks to the method.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2.3 Modeling** <span id="2.4.2.3"></span> 
**2.4.2.3.1 Bayes’ Theorem** <span id="2.4.2.3.1"></span> 
-   **Reading: Stanford Encyclopedia of Philosophy’s “Bayes’ Theorem”**
    Link: Stanford Encyclopedia of Philosophy’s [“Bayes’
    Theorem”](http://plato.stanford.edu/entries/bayes-theorem/) (HTML)  
      
     Instructions: This reading describes Bayes theorem and then
    introduces the mathematical basis for how it works and how it can be
    used.  
      
     Reading this material should take approximately 2 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2.3.2 Bayesian Networks** <span id="2.4.2.3.2"></span> 
-   **Reading: PR-OWL’s “What Is a Bayesian Network?”**
    Link: PR-OWL’s [“What Is a Bayesian
    Network?”](http://www.pr-owl.org/basics/bn.php) (HTML)  
      
     Instructions: This is an introduction to the mathematical treatment
    of Bayesian networks; it includes limitations and a case study.  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2.3.3 Likelihood Ratios** <span id="2.4.2.3.3"></span> 
-   **Reading: University of Utah’s Virtual Labs in Probability and
    Statistics: “Likelihood Ratio Tests”**
    Link: University of Utah’s Virtual Labs in Probability and
    Statistics: [“Likelihood Ratio
    Tests”](http://www.math.uah.edu/stat/hypothesis/Likelihood.html)
    (HTML)  
      
     Instructions: This reading describes likelihood ratios and provides
    several alternative tests for hypothesis testing.  Where could
    likelihood ratios be used in biology?  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**2.4.2.3.4 Clustering Coefficients** <span id="2.4.2.3.4"></span> 
-   **Reading: Wikipedia’s “Clustering Coefficient”**
    Link: Wikipedia’s [“Clustering
    Coefficient”](http://en.wikipedia.org/wiki/Clustering_coefficient)
    (HTML)  
      
     Instructions: This reading defines clustering coefficients and
    describes the different types used in graph theory.  What is the
    advantage of using one type over another?  
      
     Reading this material should take approximately 45 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.


