---
layout: default
title: "BIO405: Computational Biology"
course_description: "An introduction to the use of computers to automate data analysis or model hypotheses in the field of biology, and its application for molecular and cellular biology, biochemistry, neuroscience, evolution, population biology, and behavior."
next: ../Unit04
previous: ../Unit02
---
**Unit 3: Modeling Biological Processes** <span id="3"></span> 
*Nearly any biological process can be reduced to a mathematical formula
and modeled on a computer.  The large, multivariable datasets involved
in the study of complex processes can in fact be understood much more
quickly and easily with the help of computer modeling.  An animal
behaviorist can use mathematical prediction models to instantaneously
predict the reaction that a subject will have to a stimulus, though the
spectrum of possible outcomes and the complexity of the mathematical
model might appear quite daunting to someone attempting to model it
manually.  Alternately, using computational modeling, a biochemist could
quickly predict the variety of outcomes that a cellular cascade reaction
might have based upon the amount of substrate introduced to the cell
and, in this way, make a better estimate as to the amount that should be
used in a particular experiment or in order to achieve a particular
desired outcome.   In this unit, we will look at examples of
computational modeling in biochemistry, cell biology, neuroscience,
population biology, evolution, and behavior.  You will learn the basic
procedures for computational modeling in each of these fields and will
gain a fuller understanding of the flexibility and universal
applicability of mathematical modeling.*

**Unit 3 Time Advisory**  
This unit will take approximately 27.5 hours to complete.  
  
 ☐    Subunit 3.1: 2.5 hours  
  
 ☐    Subunit 3.2: 8 hours  
  
 ☐    Subunit 3.3: 5.5 hours  
  
 ☐    Subunit 3.4: 1 hour  
  
 ☐    Subunit 3.5: 7 hours  
  
 ☐    Subunit 3.6: 3.5 hours

**Unit3 Learning Outcomes**  
Upon completion of this unit, students will be able to:
-   Describe mathematical approaches to biochemistry and cellular
    pathways in computational biology.
-   Explain the process of neuronal signaling along with the
    mathematical components.
-   Describe how the evolutionary process can be modeled using various
    statistics.
-   Describe the mathematical treatment of population growth.

**3.1 Biochemistry Techniques** <span id="3.1"></span> 
*Note: Biochemistry is the study of the chemical reactions that occur in
biological settings.  In this section, we will learn how to model the
behavior of biologically-relevant molecules and the reaction rates of
chemical reactions catalyzed by enzymes.*

**3.1.1 Mathematical Treatment of Biochemistry** <span
id="3.1.1"></span> 
**3.1.1.1 Molecular Dynamics** <span id="3.1.1.1"></span> 
-   **Reading: Wikipedia’s “Molecular Dynamics”**
    Link: Wikipedia’s [“Molecular
    Dynamics”](http://en.wikipedia.org/wiki/Molecular_dynamics) (HTML)  
      
     Instructions: The page defines molecular dynamics and presents many
    of the details of the simulations and algorithms involved.  How can
    molecular dynamics be used to understand the movements of atoms or
    even proteins?  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.1.1.2 Enzymes and Kinetics** <span id="3.1.1.2"></span> 
-   **Reading: Wikipedia’s “Enzyme Kinetics”**
    Link: Wikipedia’s [“Enzyme
    Kinetics”](http://en.wikipedia.org/wiki/Enzyme_kinetics) (HTML)  
      
     Instructions: This entry introduces you to enzyme kinetics,
    including single and multiple substrate assays and Michaelis-Menten
    and non-Michaelis-Menten kinetics.  You should be able to describe
    the factors that influence enzyme kinetics and use the
    Michaelis-Menten equation.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2 Cellular Pathways** <span id="3.2"></span> 
*Note: Chemical reactions within cells often depend upon intricate
cascades of reactions, where an interruption of any step in the sequence
can completely skew the final product by either preventing the
production of an important protein or inhibiting the breakdown and
removal of a detrimental compound from tissue.  In this subunit, we will
learn how cellular processes can be modeled and predicted using the
computer.*

**3.2.1 Mathematical Treatment of Cellular Pathways** <span
id="3.2.1"></span> 
-   **Reading: Developmental Cell: Alex Mogilner, Roy Wollman, and
    Wallace F. Marshall’s “Quantitative Modeling in Cell Biology”**
    Link: Developmental Cell: Alex Mogilner, Roy Wollman, and Wallace F.
    Marshall’s [“Quantitative Modeling in Cell
    Biology”](http://www.math.ucdavis.edu/~mogilner/Reviews.html)
    (PDF)  
      
     Instructions: Click on the link entitled “Quantitative Modeling in
    Cell Biology” to download the PDF.  Read the article and focus on
    why and how models are used in cellular pathways and what
    mathematical approaches are involved.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2.1.1 Cellular Pathways** <span id="3.2.1.1"></span> 
-   **Reading: Nature Cell Biology: Bree B. Aldridge, John M. Burke,
    Douglas A. Lauffenburger, and Peter K. Sorger’s “Physiochemical
    Modeling of Cell Signaling Pathways”**
    The Saylor Foundation does not yet have materials for this portion
    of the course. If you are interested in contributing your content to
    fill this gap or aware of a resource that could be used here, please
    submit it here.

    [Submit Materials](/contribute/)

**3.2.2 Prediction Modeling** <span id="3.2.2"></span> 
**3.2.2.1 Ordinary Differential Equation Modeling** <span
id="3.2.2.1"></span> 
-   **Reading: George Washington University: Michael J. Coleman’s
    “Population Models with Ordinary Differential Equations”**
    Link: George Washington University: Michael J. Coleman’s
    [“Population Models with Ordinary Differential
    Equations”](http://www.gwu.edu/~mathgss/old_site/Fall2006.htm)
    (PDF)  
      
     Instructions: Scroll down and click on the link entitled “Slides
    here” to download the PDF.  This is a technical reading that
    provides examples and includes mathematical derivations using
    partial differential equations.  The goal here is to understand
    differential equations in general then apply them to single species
    models of population growth followed by multiple species models.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2.2.2 Partial Differential Equations** <span id="3.2.2.2"></span> 
-   **Reading: Nature Cell Biology: Bree B. Aldridge, John M. Burke,
    Douglas A. Lauffenburger and Peter K. Sorger’s “Physiochemical
    Modeling of Cell Signal Pathways”**
    Link: Nature Cell Biology: Bree B. Aldridge, John M. Burke, Douglas
    A. Lauffenburger and Peter K. Sorger’s [“Physiochemical Modeling of
    Cell Signal Pathways”](http://www.nrbsc.org/csm_sp2010/week2.html)
    (PDF)  
      
     Instructions: Click on the link entitled “Physiochemical Modeling
    of Cell Signal Pathways” to download the PDF.  This is a review
    article that should give you a good introduction to model design and
    mathematical modeling of biochemical pathways.  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2.2.3 Flux-Balance Analysis** <span id="3.2.2.3"></span> 
-   **Reading: Nature.com’s “Flux Balance Analysis Primer”**
    Link: Nature.com’s [“Flux Balance Analysis
    Primer”](http://www.nature.com/nbt/web_extras/supp_info/nbt0201_125/info_frame.html)
    (HTML)  
      
     Instructions: This link will introduce the basic concepts of
    Flux-Balance Analysis and the complete mathematical background
    behind it.  It includes information about metabolic pathway
    construction as well.  You may want to search for articles listed in
    the literature cited section to gain more insight into the method of
    flux-balance analysis.  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2.2.4 Analysis of Extreme Pathways** <span id="3.2.2.4"></span> 
-   **Reading: University of California, San Diego: Systems Biology
    Research Group’s “Extreme Pathway Analysis”**
    Link: University of California, San Diego: Systems Biology Research
    Group’s [“Extreme Pathway
    Analysis”](http://gcrg.ucsd.edu/Downloads/ExtremePathwayAnalysis)
    (HTML)  
      
     Instructions: Scroll down the page to find several published
    articles on extreme pathway analysis.  The goal here is to become
    familiar with extreme pathway analysis and how it is being used.
     You should be able to achieve this by reading a couple of the
    articles listed under related publications on the web page.  
      
     Studying this material should take approximately 2 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.2.4 Statistical Testing for Accuracy** <span id="3.2.4"></span> 
**3.3 Neuronal Signaling** <span id="3.3"></span> 
*Note: The process of thinking and doing is so complex, it is hard to
understand how it could be duplicated in any way.  However, it all comes
down to individual action potentials, the firing of individual neurons
within a network, and the ways in which those firings affect other
neutrons within the network.  Here, we will learn about the mathematical
modeling of action potentials (and changes in membrane potential that
may or may not create an action potential) and discover how we to
predict the behavior of a single neuron or a simple neural network based
on these models.*

**3.3.1 Mathematical Treatment of Neuronal Signaling** <span
id="3.3.1"></span> 
**3.3.1.1 The Action Potential** <span id="3.3.1.1"></span> 
-   **Reading: Bryn Mawr College: Rebecca Vandiver’s “Hodgkin-Huxley
    Model”**
    Link: Bryn Mawr College: Rebecca Vandiver’s [“Hodgkin-Huxley
    Model”](http://www.brynmawr.edu/math/people/vandiver/Math210.htm)
    (PDF)  
      
     Instructions: Click on the link entitled “Hodgkin-Huxley Model” to
    download the PDF.  This presentation covers the Hodgkin-Huxley model
    of action potentials and includes a mathematical treatment.  You
    should be able to qualitatively describe the model and how it is
    applied to action potentials.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.3.1.2 Changes in Membrane Potential** <span id="3.3.1.2"></span> 
-   **Reading: Journal of Physiology: A. V. Hill’s “A New Mathematical
    Treatment of Changes of Ionic Concentration in Muscle and Nerve
    Under the Action of Electric Currents, with a Theory as to Their
    Mode of Excitation”**
    Link: Journal of Physiology: A. V. Hill’s [“A New Mathematical
    Treatment of Changes of Ionic Concentration in Muscle and Nerve
    Under the Action of Electric Currents, with a Theory as to Their
    Mode of
    Excitation”](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1533746/)
    (PDF)  
      
     Instructions: Read this article on the mathematics associated with
    membrane potentials.  What is the Nernst Theory and how does the
    information in this reading contribute to our understanding of ion
    concentrations associated with nerve conduction?  
      
     Reading this material should take approximately 2 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.3.2 Prediction Modeling** <span id="3.3.2"></span> 
**3.3.2.1 Single-Neuron Modeling** <span id="3.3.2.1"></span> 
-   **Reading: University of Sussex: Andrew Phillipides’ “Neuronal
    Signaling in Real Neurons”**
    Link: University of Sussex: Andrew Phillipides’ [“Neuronal Signaling
    in Real
    Neurons”](http://www.informatics.sussex.ac.uk/users/andrewop/Courses/CN/CNIndex.html)
    (PPT or HTML)  
      
     Instructions: Scroll down the page and click on the Lecture 2 link.
     This presentation covers resistance, capacitances, and models
    associated with electrical impulse conduction.  Pay particular
    attention to the Hodgkin-Huxley model.  
      
     Studying this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.3.2.2 Artificial Neural Networks** <span id="3.3.2.2"></span> 
-   **Reading: learnartificialneuralnetworks.com’s “Artificial Neural
    Networks”**
    Link: learnartificialneuralnetworks.com’s [“Artificial Neural
    Networks”](http://www.learnartificialneuralnetworks.com/) (HTML)  
      
     Instructions: This comprehensive website provides everything you
    need to know about artificial networks, from an introduction to
    artificial neural networks to an explanation of how to train
    networks.  
      
     Studying this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.4 Population Biology** <span id="3.4"></span> 
*Note: Predicting population viability and growth based upon
multivariable data sets that contain information on the age and sex of
individuals, the predation rate, and limiting resource factors is done
much more efficiently on a computer than it is manually.  Here we will
learn to predict population growth in a simple population using computer
modeling.*

**3.4.1 Mathematical Treatment of Population Biology** <span
id="3.4.1"></span> 
-   **Reading: Michigan State University: Steven Vieira’s “Population
    Growth Models”**
    Link: Michigan State University: Steven Vieira’s [“Population Growth
    Models”](https://www.msu.edu/user/vieirast/popmdl.htm) (HTML)  
      
     Instructions: This page discusses several population growth models
    and the math associated with them.  Pay attention to key terms and
    the differences between exponential and logistic growth.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.4.2 Predicting Population Growth** <span id="3.4.2"></span> 
-   **Reading: Michigan State University: Steven Vieira’s “Population
    Growth Models”**
    Link: Michigan State University: Steven Vieira’s [“Population Growth
    Models”](https://www.msu.edu/user/vieirast/popmdl.htm) (HTML)  
      
     Instructions: This page discusses several population growth models
    and the math associated with them.  This is the same reading as in
    3.4.1.1.  You should focus here on mathematical formulas that allow
    you to predict future population size based on current conditions.  
      
     Reading this material should take approximately 30 minutes.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.5 Evolution** <span id="3.5"></span> 
*Note: Computer modeling comes in handy when studying the evolution of a
population, species, or group of species.  Using allelic frequency (the
frequency with which a particular gene sequence appears at a given
locus) and a mathematical calculation of the Hardy-Weinberg Equilibrium,
we can quickly see whether a population is currently evolving or not.
 By comparing the physical or molecular characteristics of different
species and mathematically sorting out the species with highest numbers
of similarities, we can create phylogenies, or “family trees” of
species, which estimate the process of evolution and divergence among
those species over millions of years.  In this section, we will learn
how to predict evolutionary change within a population and evolutionary
connection between populations using computer modeling methods.*

**3.5.1 Mathematical Treatment of Evolution** <span id="3.5.1"></span> 
**3.5.1.1 Hardy-Weinberg Equilibrium** <span id="3.5.1.1"></span> 
-   **Reading: Genetics.org: Jennifer Shoemaker, Ian Painter, and B. S.
    Weir’s “A Bayesian Characterization of Hardy-Weinberg
    Disequilibrium”**
    Link: Genetics.org: Jennifer Shoemaker, Ian Painter, and B. S.
    Weir’s [“A Bayesian Characterization of Hardy-Weinberg
    Disequilibrium”](http://www.genetics.org/cgi/content/full/149/4/2079)
    (HTML)  
      
     Instructions: This article provides a little background on
    estimating Hardy-Weinberg equilibrium and demonstrates how a
    Bayesian approach can replace other methods used to estimate
    Hardy-Weinberg equilibrium.  In addition to understanding the
    Bayesian approach mentioned in the paper, you should also have an
    understanding of Hardy-Weinberg equilibrium.  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: Jon Wakefield’s “Bayesian Methods for Examining
    Hardy-Weinberg Equilibrium”**
    Link: University of Washington: Jon Wakefield’s “[Bayesian Methods
    for Examining Hardy-Weinberg
    Equilibrium](http://faculty.washington.edu/jonno/HWbiometrics.pdf)”
    (PDF)  
      
     Instructions: This paper describes various aspects of establishing
    Hardy-Weinberg equilibrium and demonstrates how a Bayesian approach
    can be used.  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.5.1.2 Creating Phylogenies** <span id="3.5.1.2"></span> 
-   **Reading: Indiana University: ENSI/SENSI’s “Making Cladograms”**
    Link: Indiana University: ENSI/SENSI’s [“Making
    Cladograms”](http://www.indiana.edu/~ensiweb/lessons/mclad.html)
    (HTML and PDF)  
      
     Instructions: This is a well-developed lesson plan that guides you
    through the creation of a cladogram.  After reading the page, scroll
    to the bottom and view the 6 page PDF on creating a cladogram.  Work
    through this as it is the best way to learn how cladograms are
    developed.  
      
     Reading this material should take approximately 2 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.5.2 Modeling Evolutionary Change** <span id="3.5.2"></span> 
-   **Reading: University of Utah: David M. Hillis, John P. Huelsenbeck,
    and Clifford W. Cunningham’s “Application and Accuracy of Molecular
    Phylogenies”**
    Link: University of Utah: David M. Hillis, John P. Huelsenbeck, and
    Clifford W. Cunningham’s [“Application and Accuracy of Molecular
    Phylogenies”](http://www.hum.utah.edu/~mhaber/Documents/Course%20Readings/)
    (PDF)  
      
     Instructions: Click on the the link entitled
    “Hillis\_etal\_Science\_94.pdf” to download the PDF.  The paper
    discusses various models of evolution and different methods of
    creating phylogenies.  You should be able to compare the performance
    of parsimony, neighbor-joining, and UPGMA.  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.5.3 Modeling Evolutionary Connections** <span id="3.5.3"></span> 
-   **Reading: Proceedings of the National Academy of Sciences: Mónica
    Medina’s “Genomes, Phylogeny, and Evolutionary Systems Biology”**
    Link: Proceedings of the National Academy of Sciences: Mónica
    Medina’s [“Genomes, Phylogeny, and Evolutionary Systems
    Biology”](http://www.pnas.org/content/102/suppl.1/6630.full)
    (HTML)  
      
     Instructions: This article explains how genomic data has improved
    our understanding of phylogenies and also links genomic data to the
    rapidly developing field of systems biology.  You should be able to
    describe advances in biology using genomic data, systems biology,
    and transcriptional networks.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.6 Behavior** <span id="3.6"></span> 
*Note: To think that behavior can be mathematically predicted seems
counter-intuitive, yet a number of mathematical formulas and matrices
have been created and successfully deployed in order to predict the
outcomes of conflicts between animal behavior and psychology.  The study
of behavior through mathematics is famously known as “Game Theory.”  In
this section, we will learn about a few standard methods of predicting
behavior through Game Theory and discover how to model them on the
computer.*

**3.6.1 Mathematical Treatment of Behavior** <span id="3.6.1"></span> 
**3.6.1.1 Game Theory** <span id="3.6.1.1"></span> 
-   **Reading: University of California, Los Angeles: David K. Levine’s
    “What Is Game Theory?”**
    Link: University of California, Los Angeles: David K. Levine’s
    [“What Is Game
    Theory?”](http://levine.sscnet.ucla.edu/general/whatis.htm#Some%20Instructive%20Examples)
    (HTML)  
      
     Instructions: This is a good introductory reading on the subject of
    game theory.  It includes an instructive example and some simple
    math behind the prisoners dilemma.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.6.1.2 Prisoner's Dilemma** <span id="3.6.1.2"></span> 
-   **Reading: University of Iowa Math Club: Erin Pearse’s “The
    Prisoner’s Dilemma”**
    Link: University of Iowa Math Club: Erin Pearse’s [“The Prisoner’s
    Dilemma”](http://math.ou.edu/~epearse/) (PDF)  
      
     Instructions: Click on the link entitled “The Prisoner's Dilemma.
    Invited speaker. April 23, 2009. Grinnell College.” to download the
    PDF.  This is a presentation that covers the game, the mathematics
    behind the game, and variations on the theme.  Pay attention to
    definitions and to real world application of the Prisoner’s
    Dilemma.  
      
     Reading this material should take approximately 1.3 hours.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**3.6.2 Prediction Modeling** <span id="3.6.2"></span> 
**3.6.2.1 Collective Behavior** <span id="3.6.2.1"></span> 
-   **Reading: Trends in Cognitive Science: Robert L. Goldstone and
    Marco A. Janssen’s “Computational Models of Collective Behavior”**
    Link: Arizona State University: R. Goldstone and M. Janssen’s
    [“Computational Models of Collective
    Behavior”](http://www.public.asu.edu/~majansse/) (PDF)  
      
     Instructions: Scroll down to the line “70. Goldstone, R.L. and M.A.
    Janssen (2005) Computational models of collective behaviour, Trends
    in Cognitive Science 9(9): 424-430” and click the link to download
    the PDF.  You should be able to compare and contrast the different
    Agent-Based models from the reading.  
      
     Reading this material should take approximately 1 hour.  
      
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.


